### ISSUE TITLE
Unable to get broker/topic info when JMX is enabled

### ISSUE THREAD

**Issue by montana-ua** [yahoo/kafka-manager#229]
Some times ago I have faced the issue "Ask timed out on [ActorSelection[Anchor(aka://kafka-manager-system/), Path(/user/kafka-manager/)]] after 5000ms" on two different kafka-manager instances. My zookeeper.connect string was correct. I had able to connect to kafka-manager web UI. I had able to select cluster and see the list of available broker but metric values were empty for it. I have checked an application.conf, network connectivity and did find any problem. I have restarted kafka-manager instance and problem has not been solved. The issue went out when I performed disable/enable JMX polling for cluster (now I can't reproduce this issue but I faced it some times on different kafka-manager instances). May be there is the bug in kafka-manager?

OS: RHEL 6.7
Java: 1.8.0.77
Kafka-Manager: 1.3.0.4/1.3.0.7
Zookeeper 3.4.8 (java 1.7.0.80)
Kafka 0.8.2.2 (java 1.7.0.80)

Below are my config file, application log with debug level (from the start point to first "Ask timed out" error), several screenshots.
application.conf - [https://onedrive.live.com/redir?resid=F5568011798EDE8!46246&authkey=!AMaXZwRirNR0bNs&ithint=file%2cconf](url)

**Comment by patelh** [comment]
Did you apply the patch to get broker and topic size info ?

**Comment by montana-ua** [comment]
Yes. The patch KAFKA-1614 was applied.

**Comment by userBD** [comment]
Hi did you find a solution ?

**Comment by montana-ua** [comment]
No, I did not find a solution. My workaround is disable/enable JMX for cluster.
